Source/AliveLibAE/VGA.cpp

```diff
@@ -156,20 +156,20 @@ EXPORT void CC VGA_CopyToFront_4F3730(Bitmap* pBmp, RECT* pRect, int /*screenMod
 
             int w = 0;
             int h = 0;
-            SDL_GetWindowSize(Sys_GetHWnd_4F2C70(), &w, &h);
+            SDL_GL_GetDrawableSize(Sys_GetHWnd_4F2C70(), &w, &h);
```

Comment:
we're not using GL to render so I'm not sure this is correct? GL might get picked as the backend but it could be something else...